scripts/update-readme.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs')
+const path = require('path')
+const readme = path.join(__dirname, '/../README.md')
+const data = fs.readFileSync(readme, 'utf-8')
+const distSize = fs.statSync(path.join(__dirname, 'build', '../../build/postmate.min.js')).size
+const updated = data.replace(
+  /<span class="size">(.*?)<\/span>/,
+  `<span class="size">\`${(distSize / 1024).toFixed(1)}kb\`</span>`
```

Comment:
as mentioned in the other PR - I'd suggest measuring gzipped sizes

and do that for es (or cjs) file, umd contains some extra bytes that won't be used by majority of users who use modern bundling that understands es files thanks to `module` entry in pkg.json